Utils/tests/check_pack_and_request_review_test.py

```diff
@@ -22,7 +22,51 @@ def test_check_reviewers(mocker, pr_author, version, reviewers, call_count):
     """
     check_reviewers_mock = mocker.patch('Utils.request_contributor_review.tag_user_on_pr')
 
-    check_reviewers(reviewers=reviewers, pr_author=pr_author, version=version,
-                    modified_files=['Pack/TestPack/file1'], pack='TestPack', pr_number='1', github_token='github_token',
-                    verify_ssl=True)
+    notified = check_reviewers(reviewers=reviewers, pr_author=pr_author, version=version,
+                               modified_files=['Pack/TestPack/file1'], pack='TestPack', pr_number='1',
+                               github_token='github_token',
+                               verify_ssl=True)
     assert check_reviewers_mock.call_count == call_count
+    assert notified == expected
+
+
+def test_send_email_to_reviewers(mocker, capsys):
+    """
+       Given
+        - reviewers_emails - developer mails to review the changes on the pack
+        - api_token - sendgrid api token in order to obtain access key
+        - pack_name - pack that was modified
+        - pr_number - github pr number
+
+       When
+       - calling send_email_to_reviewers function
+
+       Then
+       - validating that message was sent sucessfully, and information printed out
+    """
+
+    class SgMock:
+        def __init__(self):
+            self.client = self
+            self.mail = self
+            self.send = self
+            self.status_code = 202
+
+        def post(self, request_body):
+            return self
```

Comment:
I know what you did works but it still a bit unclear, let's do this:
 ```suggestion
    # A mocker for the sendgrid library
    class SgMock:
        def __init__(self):
            self.client = self.Client()

        class Client:
            def __init__(self):
                self.mail = self.Mail()

            class Mail:
                def __init__(self):
                    self.send = self.Send()

                class Send:
                    def post(self, request_body):
                        return self.Res(request_body)

                    class Res:
                        def __init__(self, _):
                            self.status_code = 202

```